### ISSUE TITLE
[polaris] Ingress spec is not valid

### ISSUE THREAD

**Issue by js-timbirkett** [FairwindsOps/charts#519]
**Describe the bug**
PR #489 breaks installation on clusters supporting `networking.k8s.io/v1`. Looks like a little more work is needed than just changing the `apiVersion` :D

**To Reproduce**
1. Try upgrading using this Helm chart on a modern version of Kubernetes. 

**Expected behavior**
No errors for invalid resource specs.

**CLI Output**
```
STDERR:
  Error: Failed to render chart: exit status 1: coalesce.go:160: warning: skipped value for config: Not a table.
  Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: [ValidationError(Ingress.spec.rules[0].http.paths[0].backend): unknown field "serviceName" in io.k8s.api.networking.v1.IngressBackend, ValidationError(Ingress.spec.rules[0].http.paths[0].backend): unknown field "servicePort" in io.k8s.api.networking.v1.IngressBackend]
  Error: plugin "diff" exited with error
  ```
**Environment (please complete the following information):**
 - Helm Version: 3
 - Kubernetes Version 1.20

**Additional context**
See: https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource

**Comment by swade1987** [comment]
I noticed the same thing when upgrading to 1.19 on the latest 4.0.6 chart version.

**Comment by rbren** [comment]
Hi @js-timbirkett @swade1987 - thanks for reporting! Apologies for the delay in fixing this.

I have a candidate solution here: https://github.com/FairwindsOps/charts/pull/545 - can you give that a shot?

**Comment by rbren** [comment]
Fixed in 4.0.7 - thanks all
